django/db/migrations/operations/base.py

```diff
@@ -1,6 +1,17 @@
+import enum
+
 from django.db import router
 
 
+class OperationCategory(str, enum.Enum):
+    ADDITION = "+"
+    REMOVAL = "-"
+    ALTER = "~"
```

Comment:
One thing that bothers me (and perhaps only me) is that we have `ADDITION`, `REMOVAL`, then `ALTER`. I feel it should be `ALTERATION` in keeping with the others.